news: No more waitlist (#124)
FlowFuse/website: src/blog/2022/05/sign-up-for-flowforge-cloud.md

---
title: FlowForge open for everybody
subtitle: Sign up and start a new Node-RED project within a minute!
description: With the waitlist now phased out users can sign up and start using FlowForge right away and start developing on new projects.
date: 2022-05-25 09:00:00.0
authors: ["zeger-jan-van-de-weg"]
---

> FlowForge wants to enable everyone to build workflows in Node-RED. Since announcing
> [FlowForge Cloud](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/)
> two months ago we've employed a waitlist to was employed to control the pace the
> new users to our platform with which our team could learn what is needed to
> scale up our platform, and create a good first use experience with FlowForge.
> 
> Today, anyone can sign up to FlowForge and start a new Node-RED project in under
> a minute!

Reviewer: ```suggestion
FlowForge wants to enable everyone to build workflows in Node-RED. Since announcing
[FlowForge Cloud](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/)
two months ago we've had a waiting list for users to sign up to. That allowed us
to control the pace we were bringing new users onto the platform, learning what
is needed to scale up our platform and continue to improve our first user experience.

Today we have removed the waiting list. Anyone can sign up to FlowForge and start a
new Node-RED project in under a minute!
```

<!--more-->

<div class="max-w-md m-auto">
  <a class="ff-btn ff-btn--primary" href="https://app.flowforge.com/account/create">Sign up</a>
</div>

## What we offer

Besides the sub minute time to start a new Node-RED project, there's many more
features our offering includes. Two we'd like to highlight.

To start our blog highlight reel: Collaboration. FlowForge allows you to work
with a team on your flows. There's the ability to create multiple users, each
with their own credentials that can alter the flows on Node-RED, and it's
execution environment like for example the [environments variables](https://flowforge.com/docs/user/envvar/).

> Furthermore, [stacks](https://flowforge.com/docs/user/changestack). These allow
> a user to select the execution environment for the Node-RED. For example; the
> Node-RED version being used. Combined with the ability for one to copy a project
> to a new stack this allows FlowForge users to copy their project to the Node-RED
> 3.0-beta stack to validate their solutions will continue to work if a stable
> release of Node-RED is done. When validation is done, the beta project can be
> deleted after and the main projects stack can be upgraded.

Reviewer: ```suggestion
Furthermore, [stacks](https://flowforge.com/docs/user/changestack). These allow
a user to select the execution environment for their Node-RED project. For example; the
Node-RED version being used. Combined with the ability for one to copy a project
to a new stack, this allows FlowForge users to copy their project to the Node-RED
3.0-beta stack to validate their solutions will continue to work on the new release
without disrupting their main project.
```

## Our roadmap

> Currently we're working towards our 0.6 release. The main feature of this release
> will be support for Devices. This would allow you to send a snapshot of a project
> to Node-RED instances outside of the FlowForge instance and update the flows
> remotely. Remote devices will run on an [agent](https://github.com/flowforge/flowforge-device-agent)
> to communicate with the FlowForge Cloud project. While the first iteration would
> be labeled an Alpha release, we welcome feedback and anticipate it would allow
> customers to validate the offering in new proof of concept projects.

Reviewer: ```suggestion
Currently we're working towards our 0.6 release. The main feature of this release
will be support for Devices. This will allow you to send a snapshot of a project
to a Node-RED instance running outside of the FlowForge platform and update the flows
remotely. Remote devices will run our [agent](https://github.com/flowforge/flowforge-device-agent)
to communicate with the FlowForge Cloud project.

While the first iteration will be considered an Alpha release, by shipping early and often, it lets us get
welcome feedback from our users and the wider community - helping to shape the future direction.
It also allows users to start validating the feature for their own proof of concept projects.
```

> In the next months we're continuing our development around devices. Further, we
> intent to grow our offering so that FlowForge remains the best way to run
> Node-RED, introduce new enterprise features like Single-Sign on, and make
> collaboration with Node-RED even easier than today.

Reviewer: ```suggestion
Over the next few months we're continuing to drive development of the platform
across a number of areas - including further improvements to the Device feature.
But also looking at new Enterprise-ready features, such as Single-Sign On integration
and more tools to make collaboration even easy.

We intend to grow our offering so that FlowForge remains the best way to run
Node-RED.
```
